src/main/java/com/hibernate/model/MovieSession.java

```diff
@@ -22,6 +22,7 @@
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "hall_id")
     private CinemaHall cinemaHall;
+    @JoinColumn(name = "show_time")
```

Comment:
you are right - I misspelled, you should better use `@Column` for this purpose if you do not have relations
` @Column(name = "show_time")`